pkg/minikube/cluster/cluster.go

```diff
@@ -112,6 +113,18 @@ func StartHost(api libmachine.API, config cfg.MachineConfig) (*host.Host, error)
 	s, err := h.Driver.GetState()
 	glog.Infoln("Machine state: ", s)
 	if err != nil {
+		// If VirtualBox Machine does not exist due to user interrupt cancel(i.e. Ctrl + C), recreate VirtualBox Machine
+		if driver.BareMetal(config.VMDriver) || config.VMDriver == driver.VirtualBox {
```

Comment:
could you please remind me why we need this check for none driver ?
``` driver.BareMetal(config.VMDriver)``` ?

Reply:
Thanks for your point out.
I thought that Mock driver judgement is needed and `driver.BareMetal` is its method.
But I misunderstood.

I modified this `driver.BareMetal` → `config.VMDriver == driver.Mock`.